src/Components/CartCard/CartCard.css

```diff
@@ -1,134 +1,144 @@
 .cart-page {
-    display: flex;
-    max-width: 61.25rem;
-    margin: 0 auto 2rem;
-    margin-left: 2rem;
-    top: 10;
-    left: 0;
-    right: 0;
-    z-index: 10;
+  display: flex;
+  max-width: 61.25rem;
+  margin: 0 auto 2rem;
+  margin-left: 2rem;
+  top: 10;
+  left: 0;
+  right: 0;
+  z-index: 10;
 }
 
 .cart-container {
-    display: flex;
-    flex: 0 0 70%;
+  display: flex;
+  flex: 0 0 70%;
 }
 
 .cart-head {
-    padding: 1rem 1rem;
-    font-size: 1rem;
-    letter-spacing: 1.5px;
-    display: flex;
-    justify-content: flex-start;
-    align-items: center;
-    white-space: nowrap;
-    gap: 1rem;
+  padding: 1rem 1rem;
+  font-size: 1rem;
+  letter-spacing: 1.5px;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  white-space: nowrap;
+  gap: 1rem;
 }
 
 .cart-card {
-    width: 100%;
-    background-color: var(--main-background-color);
-    border-radius: 2px;
-    margin-bottom: 4rem;
-    margin-top: 4rem;
-    display: flex;
-    flex-direction: column;
-    gap: 2rem;
+  width: 100%;
+  background-color: var(--main-background-color);
+  border-radius: 2px;
+  margin-bottom: 4rem;
+  margin-top: 4rem;
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
 }
 
 /* HORIZONTAL CARD CSS  */
 .cart-card .card-container {
-    width: 40rem;
-    display: flex;
-    flex-direction: column;
-    padding-bottom: 0.5rem;
-    gap: 0.5rem;
-    position: relative;
-    border-radius: 0.4rem;
-    background-color: var(--card-BG);
-    box-shadow: 4px 4px 15px rgba(0, 0, 0, 0.15);
+  width: 40rem;
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 0.5rem;
+  gap: 0.5rem;
+  position: relative;
+  border-radius: 0.4rem;
+  background-color: var(--card-BG);
+  box-shadow: 4px 4px 15px rgba(0, 0, 0, 0.15);
 }
 
 .card-container .image-box {
-    height: 12.5rem;
-    width: 10rem;
+  height: 12.5rem;
+  width: 10rem;
 }
 
 .card-container .card-details p,
 h4 {
```

Comment:
suggestion: - use className for styling heading... else it may conflict with another heading if u don't want to style them.